entities/Receipt.java

```diff
@@ -10,15 +11,19 @@
  * */
 
 public class Receipt {
+
+    private static int totalReceipts;
```

Comment:
The idea behind this was, since there are multiple billing counters and each would generate some new receipts, there could be a counter to keep track of. I could do getReceiptsCount()  in Shop where i can iterate through all billing counters to get the total number of receipts generated in the Shop.  Please let me know some scenarios where i could use the static variables other than constants.

Reply:
Understood. Thank you.